javascript/ql/src/Declarations/SuspiciousMethodName.ql

```diff
@@ -0,0 +1,80 @@
+/**
+ * @name Suspicious method name
+ * @description A method having the name "function", "new", or "constructor"
```

Comment:
The `@description` is usually of the form `Doing X, causes Y`, rather than your `Doing X is caused by Y`. 
Could you reformulate along the lines of "Declaring a class or interface member with a special name may cause unexpected types to be declared" (needs a bit more polish).

Reply:
`Declaring a class or interface method with a special name may cause a normal named method to be declared when a special type was expected.`?